library/ix-dev/community/homepage/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+runAsContext:
+  - userName: homepage
+    groupName: homepage
+    gid: 1000
```

Comment:
I've tried 568, it was mostly working, but when it tried to create some pre-rendered content it was failing to do.
But I have something in mind to try, I'll look into it

Reply:
Only user/group 1000 have write access to app directories.

While the "error" comes from trying to write in directories that are not persistent, and used for cache, we cannot put them on a volume (eg emptyDir) because those are pre-populated with already cached files among other built artifacts that are not regenerated on the fly.